text-speech/texttospeech.java

```diff
@@ -0,0 +1,58 @@
+import com.sun.speech.freetts.*;
+import java.io.* ;
+public class texttospeech
+{
+private String speaktext;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'speaktext' could be replaced by a local variable.](https://www.codacy.com/app/maratmingazovr/Java/file/2724722624/issues/source?bid=3348317&fileBranchId=3348330#l5)